### PR TITLE
[fix] introduce helper method to prevent push maximum call stack size exceeded error

### LINKED ISSUE #40: Array push spread v8
Hi,
Coming from https://github.com/sveltejs/svelte/issues/4694.
`handlers.ts` is using array push with spread operator in multiple places and it is causing max call stacks in v8 (too many params in a function call).
The issue is providing an example but I can come up with a repro if you want.
I'd like to handle it but I am not sure what you prefer: using for loops, array concat, or even using a codemod ? Or not handling it at all 😄 
Cheers,

### PR DESCRIPTION
**Pull request by dummdidumm**
As discussed in the maintainers meeting, this just introduces a simple `push_array` method

Fixes #40
Alternative to / closes #64
Part of sveltejs/svelte#4694

Credit to @milahu for finding the issue, we should add a "co-authored by"-message to the merge.

### CODE REVIEW

File: src/utils/push_array.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * "Maximum call stack size exceeded" when items
+ * is too big of an array
+ * @param {any[]} array 
+ * @param {any[]} items 
+ */
+ export function push_array(array, items) {
+    for (let i = 0; i < items.length; i++) {
+      array.push(items[i]);
+    }
+  };
```
Comment (benmccann) [inline src/utils/push_array.js:13]: it looks like there's whitespace before the `}` which shouldn't be necessary. same above before the `export`

File: src/utils/push_array.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Does `array.push` for all `items`.
+ * Needed because `array.push(...items)` throws
+ * "Maximum call stack size exceeded" when items
+ * is too big of an array
```
Comment (benmccann) [inline src/utils/push_array.js:5]: this seems like a pretty short line wrap length

**Review by benmccann — COMMENTED** [review]
lgtm. thanks for tackling this!

File: src/utils/push_array.js (hunk 2 above)
Comment (dummdidumm) [reply src/utils/push_array.js]: Adjusted 👍

File: src/utils/push_array.js (hunk 1 above)
Comment (dummdidumm) [reply src/utils/push_array.js]: I'm no longer able to format code since using Prettier 😄 Fixed

**Review by benmccann — APPROVED** [review]

**Pull request merged**